.gitignore

```diff
@@ -8,3 +8,4 @@
 /spec/reports/
 /tmp/
 activerecord-mysql-uuid-column-*.gem
+/.idea/
```

Comment:
I am going to accept this, but just to mention my two cents here, since this is not project specific but environment specific, my recommendation is to add this to your personal gitignore file 

```
$ cat ~/.gitconfig 
[user]
        email = ...
        name = ...
        signingkey = ...
[push]
        default = simple
[core]
        excludesfile = /home/.../.gitignore_global
```